src/Collapse.tsx

```diff
@@ -81,6 +82,7 @@ const Collapse = React.forwardRef<HTMLDivElement, CollapseProps>((props, ref) =>
       className={collapseClassName}
       style={style}
       role={accordion ? 'tablist' : undefined}
+      {...restProps}
```

Comment:
https://github.com/ant-design/ant-design/blob/31ff625d8f9138dcb82079223ac7c255358b9d91/components/radio/group.tsx#L110

Pick data-* and aria-* only.

Reply:
Sure! Thank you, done